cli/src/cmd/init.js

```diff
@@ -30,6 +32,16 @@ export async function init() {
       name: 'projectName',
       message: `What's your project name? ${color.promptHelper('(A folder with this name will be created)')}`,
       initial: targetDir,
+      validate: (value) => {
+        try {
+          projectNameSchema.parse(value);
+          return true;
+        } catch (e) {
+          return fromError(e)
+            .details.map((issue) => issue.message)
+            .join('; ');
```

Comment:
I would have preferred something simpler but didn't manage to achieve it without printing `Validation error`. Open for suggestions